Documentation/ceph-openshift.md

```diff
@@ -3,86 +3,44 @@ title: OpenShift
 weight: 2060
 indent: true
 ---
+
 {% include_relative branch.liquid %}
 
 # OpenShift
 
 [OpenShift](https://www.openshift.com/) adds a number of security and other enhancements to Kubernetes. In particular, [security context constraints](https://blog.openshift.com/understanding-service-accounts-sccs/) allow the cluster admin to define exactly which permissions are allowed to pods running in the cluster. You will need to define those permissions that allow the Rook pods to run.
 
-The settings for Rook in OpenShift are described below, and are also included in the [example yaml files](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph):
+The settings for Rook in OpenShift are described below, and are also included in the [example yaml files](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/openshift/ceph):
```

Comment:
While it's really temping to move the openshift examples to an openshift directory, this is also awkward because then you still have to use the kubernetes directory to create pools or other CRs that don't have openshift-specific settings. We could symlink all the other yamls too, but that's tedious. For this reason I think it actually works better to keep them in the same folder with all the yamls and have the openshift in the filename. 

Now that the storage providers are all in separate repos, what really seems better is if we put all the yamls in `cluster/examples/`, and no need for a `kubernetes` or `ceph` subfolder. But that change would also probably break a lot of people's automation so I'm not sure it's worth the change.

Reply:
> While it's really temping to move the openshift examples to an openshift directory, this is also awkward because then you still have to use the kubernetes directory to create pools or other CRs that don't have openshift-specific settings. We could symlink all the other yamls too, but that's tedious. For this reason I think it actually works better to keep them in the same folder with all the yamls and have the openshift in the filename.

All of that makes sense and it's tempting I agree. It's convenient but wrong to have it under the `kubenertes` folder.

> 
> Now that the storage providers are all in separate repos, what really seems better is if we put all the yamls in `cluster/examples/`, and no need for a `kubernetes` or `ceph` subfolder. But that change would also probably break a lot of people's automation so I'm not sure it's worth the change.

I was thinking the same but didn't make that change for that exact reason. Although I've always had an issue with that long path to remember ^^. Honestly, I just wish everything would go under an `example` folder at the root of the repo.
Honestly, for the sake of clarity, I'd be willing to take that action even if this breaks some automation. It's easy to fix anyways.

Reply:
I am **VERY** into just having examples. `charts` should be separate anyway. 

We could also create some symlinks that could help ease the transition for peoples' automation.

Reply:
Agreed, simply `examples` would be nice. As long as we keep it as part of a minor release (v1.8), it seems like a reasonably small breaking change.

Reply:
proposed change https://github.com/rook/rook/pull/8740